tex/alg-NT/frobenius.tex

```diff
@@ -346,22 +354,13 @@ \section{Frobenius elements behave well with restriction}
 	i.e.\ for every $\alpha \in K$,
 	\[ \Frob_\kp(\alpha) = \Frob_{\kP}(\alpha). \]
 \end{theorem}
-%\begin{proof}
-%	We know
-%	\[ \Frob_{\kP}(\alpha) \equiv \alpha^p \pmod{\kP}
-%		\quad \forall \alpha \in \OO_L \]
-%	from the definition.
-%	\begin{ques}
-%		Deduce that
-%		\[ \Frob_{\kP}(\alpha) \equiv \alpha^p \pmod{\kp}
-%			\quad \forall \alpha \in \OO_K. \]
-%		(This is weaker than the previous statement in two ways!)
-%	\end{ques}
-%	Thus $\Frob_{\kP}$ restricted to $\OO_K$ satisfies the
-%	characterizing property of $\Frob_\kp$.
-%\end{proof}
 \begin{proof}
-	TODO: Broken proof. Needs repair.
+	First, $K/\QQ$ is normal, so $\Frob_{\kP}$ fixes the image of $K$, that is,
+	$\Frob_{\kP} \restrict{K} \in \Gal(K/\QQ)$ is well-defined.
+
+	We have the natural map $\phi \colon \OO_K \to \OO_L \to \OO_L/\kP$, and the quotient map $q\colon \OO_K \to \OO_K / \kp$. Since $\kP \subseteq \kP$, then $\phi$ factors through $q$ to give a natural field homomorphism $\OO_K / \kp \to \OO_L / \kP$.
```

Comment:
Seem like an interesting side result, but I don't know if this is useful in other ways.

Reply:
Only after looking at it compiled I realize I made a mistake, it's supposed to say 𝔭 ⊆ 𝔓. (but the more correct thing to say would be 𝔭 ⊆ 𝔓 ∩ 𝒪ₖ ⊆ ker φ, which is the reason why φ factors through q)